Analysis/Tasks/PWGHF/HFTrackIndexSkimsCreator.cxx

```diff
@@ -62,9 +65,7 @@ struct SelectTracks {
       if (abs(track.eta()) > etamax_3prong)
         status_3prong = 0;
       UChar_t clustermap = track.itsClusterMap();
-      bool isselected = track.tpcNClsFound() >= d_tpcnclsfound &&
-                        track.flags() & o2::aod::track::ITSrefit &&
-                        (TESTBIT(clustermap, 0) || TESTBIT(clustermap, 1));
+      const bool isselected = track.isGlobalTrack();
       if (!isselected) {
```

Comment:
You can also directly use a filter to select these tracks. 
Filter trackFilter =((aod::track::isGlobalTrack == true)